backstage/packages/app/src/scaffolder/GithubTeamPicker/schema.ts

```diff
@@ -1,15 +1,15 @@
 import { z } from 'zod';
 import { makeFieldSchemaFromZod } from '@backstage/plugin-scaffolder';
+import { makeFieldSchema } from '@backstage/plugin-scaffolder-react';
 
-export const GithubTeamPickerFieldSchema = makeFieldSchemaFromZod(
-  z.string(),
-  z.object({
-    title: z.string().default('Github Team').describe('Github Team'),
-    desciption: z
-      .string()
-      .default('Select a Github Team')
-      .describe('Select a Github Team'),
-  }),
+const output = (zImpl: typeof z) => zImpl.string().min(1);
```

Comment:
like the previous commit, I'd really like to understand what motivated the refactor.

Reply:
updated commit message